packages/core/src/routes/admin-user.ts

```diff
@@ -27,18 +27,23 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
     '/users',
     koaPagination(),
     koaGuard({
-      query: object({ search: string().optional(), hideAdminUser: literal('true').optional() }),
+      query: object({
+        search: string().optional(),
+        hideAdminUser: literal('true').optional(),
+        isCaseSensitive: z.enum(['true', 'false']).default('true'),
```

Comment:
Should set default to `false`, IMO

Reply:
Can align this guard with `hideAdminUser`: 

```ts
isCaseSensitive: literal('true').optional(),
```

Reply:
I've updated